### PR TITLE
Add test case for changing passwords

### PR DESCRIPTION
**Pull request by rhcarvalho**
This is a follow up to #50 

@mfojtik @bparees PTAL

### CODE REVIEW

File: 9.2/test/run [hunk 1]
```diff
@@ -212,6 +212,71 @@ function run_tests() {
...
+  test_connection ${name}
+
+  echo "  Testing login"
+  assert_login_access ${user} ${password} true
+  USER='postgres'
+  PASS=${admin_password}
+  assert_login_access 'postgres' ${admin_password} true
+
+  echo "  Changing passwords"
+  docker stop $(get_cid ${name})
```
Comment (rhcarvalho) [inline 9.2/test/run]: For some reason my local tests fail after this, and I cannot start the new container getting:

> FATAL:  configuration file "/var/lib/pgsql/data/userdata/postgresql.conf" contains errors

Quickly glanced the file and it seemed ok, though.

File: 9.2/test/run [hunk 2]
```diff
@@ -212,6 +212,71 @@ function run_tests() {
...
+
+  # need this to wait for the container to start up
+  CONTAINER_IP=$(get_container_ip "${name}_NEW")
+  test_connection "${name}_NEW"
+
+  echo "  Testing login with new passwords"
+  assert_login_access ${user} "NEW_${password}" true
+  USER='postgres'
+  PASS="NEW_${admin_password}"
+  assert_login_access 'postgres' "NEW_${admin_password}" true
```
Comment (bparees) [inline 9.2/test/run]: make sure the old password is invalid, too?

File: 9.2/test/run [hunk 3]
```diff
@@ -212,6 +212,71 @@ function run_tests() {
...
+  docker stop $(get_cid ${name})
+  DOCKER_ARGS="
+-e POSTGRESQL_DATABASE=${database}
+-e POSTGRESQL_USER=${user}
+-e POSTGRESQL_PASSWORD=NEW_${password}
+-e POSTGRESQL_ADMIN_PASSWORD=NEW_${admin_password}
+-v ${volume_dir}:/var/lib/pgsql/data
+" create_container "${name}_NEW"
+
+  # need to set these because `postgresql_cmd` relies on global variables
```
Comment (rhcarvalho) [inline 9.2/test/run]: This is ugly, but needed for now before I go on and fix existing code (we, in fact, never test the admin password, but test the user password all the time).

File: 9.2/test/run (hunk 2 above)
Comment (rhcarvalho) [inline 9.2/test/run]: I'd say in general if one password works, the other, by exclusion, doesn't. But in this reality we have here testing both can make sense...

I'm almost sure `assert_login_access` always return 0, though. So all those tests always pass...

Comment (rhcarvalho) [inline 9.2/test/run]: Now we check new and old passwords

**Comment by jhadvig** [comment]
Shouldn't we step away from the bash test monster that we are slowly feeding in our images and rather focus on rewriting them to extended-tests, since we finally got them ?

**Comment by bparees** [comment]
@jhadvig yes but first we'd need to actually change our image test scripts to run the extended tests on openshift.  running the extended tests in origin doesn't help us test image repo PRs.

so i think we're still a bit away from converting everything into extended tests.

**Comment by bparees** [comment]
lgtm.

**Pull request merged**